tools/lib/logreader.py

```diff
@@ -333,7 +333,15 @@ def time_series(self):
   # capnproto <= 0.8.0 throws errors converting byte data to string
   # below line catches those errors and replaces the bytes with \x__
   codecs.register_error("strict", codecs.backslashreplace_errors)
-  log_path = sys.argv[1]
-  lr = LogReader(log_path, sort_by_time=True)
-  for msg in lr:
-    print(msg)
+
+  parser = ArgumentParser(description="Process a log file and print identifiers or full messages.")
+  parser.add_argument("log_path", help="Path to the log file")
+  parser.add_argument("--identifiers-only", action="store_true", help="Print only log identifiers")
+  args = parser.parse_args()
+
+  lr = LogReader(args.log_path, sort_by_time=True)
+  if args.identifiers_only:
+    print("\n".join(lr.logreader_identifiers))
+  else:
+    for msg in lr:
+      print(msg)
```

Comment:
Can you make this a script inside cabana/? I don't like polluting this for something specific to Cabana.

I still think a Python cabana wrapper that passes opts into C++ Cabana might be simpler

Reply:
moved to tools/cabana/auto_source.py